hibike/lib/hibike/hibike_device.cpp

```diff
@@ -120,17 +134,23 @@ void hibike_loop() {
   }
 
   // DataUpdates
+  // if ((subDelay > 0) && (currTime - prevTime >= subDelay)) {
```

Comment:
Delete instead of commenting out. Add TODO if this functionality will need to be re-enabled later.